.travis.yml

```diff
@@ -34,168 +34,202 @@ jobs:
 
     # Filebeat
     - os: linux
+      before_install: if bash dev-tools/travis_skip_changes.sh $TRAVIS_COMMIT_RANGE libbeat filebeat vendor; then echo "NOT testing"; travis_terminate 0; fi
       env: TARGETS="-C filebeat testsuite"
       go: $TRAVIS_GO_VERSION
       stage: test
     - os: osx
+      before_install: if bash dev-tools/travis_skip_changes.sh $TRAVIS_COMMIT_RANGE libbeat filebeat vendor; then echo "NOT testing"; travis_terminate 0; fi
```

Comment:
The first four put vendor at the end. Just to be consistent consider matching these four up to be `libbeat vendor <beat>"